src/samples/StaticPages/ChooseClaimService/index.tsx

```diff
@@ -0,0 +1,117 @@
+      label: t('ADD_CHILD_TO_EXISTING_CLAIM')
+    },
+    {
+      value: 'checkonprogressofclaim',
+      label: t('CHECK_PROGRESS_OF_CLAIM')
+    },
+    {
+      value: 'restartyourpayments',
+      label: t('RESTART_YOUR_PAYMENTS')
+    },
+    {
+      value: 'stopyourpayments',
+      label: t('STOP_YOUR_PAYMENTS')
+    }
+  ];
+
+  function clicked() {
+    const selectedOption = document.querySelector('input[name="serviceType"]:checked');
+    if (selectedOption) {
+      const selectedOptionValue = selectedOption.getAttribute('value');
+      switch (selectedOptionValue) {
+        case 'makeanewclaim':
+          history.push('/accessibility');
+          break;
+        case 'addchildtoexistingclaim':
+          history.push('/cookies');
+          break;
+        case 'checkonprogressofclaim':
+          break;
+        case 'restartyourpayments':
+          window.location.assign(
+            'https://www.gov.uk/child-benefit-tax-charge/restart-child-benefit'
+          );
+          break;
+        case 'stopyourpayments':
+          window.location.assign('https://www.gov.uk/child-benefit-tax-charge/stop-child-benefit');
+          break;
+        default:
+          break;
+      }
+    } else {
+      setErrorMsg(t('CONFIRM_YOUR_SERVICE'));
+    }
+  }
+  return (
+    <>
+      <AppHeader appname={t('CLAIM_CHILD_BENEFIT')} hasLanguageToggle isPegaApp={false} />
+      <div className='govuk-width-container'>
+        <Button
+          variant='backlink'
+          onClick={() => history.goBack()}
+          key='ChooseClaimBacklink'
+          attributes={{ type: 'link' }}
+        />
+        <MainWrapper>
+          {errorMsg && errorMsg.length > 0 && (
+            <div className='govuk-error-summary' data-module='govuk-error-summary' tabIndex={-1}>
+              <div role='alert'>
+                <h2 className='govuk-error-summary__title'>There is a problem</h2>
+                <div className='govuk-error-summary__body'>
+                  <ul className='govuk-list govuk-error-summary__list'>
+                    <li key='serviceType'>
+                      <a href={`#serviceType`}>{errorMsg}</a>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          )}
+          <RadioButtons
+            name='serviceType'
+            displayInline={false}
+            value=''
+            useSmallRadios={false}
+            options={radioOptions}
+            label={t('MAKE_SURE_YOU_USE_RIGHT_CHBS')}
+            legendIsHeading
+            hintText={`<p class="govuk-body">${t('CONFIRM_YOUR_SERVICE')}</p>`}
+            errorText={errorMsg}
+          ></RadioButtons>
+          <button
+            className='govuk-button'
+            data-module='govuk-button'
+            onClick={clicked}
+            type='button'
+          >
+            {t('CONTINUE')}
+          </button>
+          <br></br>
+          <br></br>
```

Comment:
add padding

Reply:
we can't use margin bottom here . I have tried doing that , not working.

Reply:
This is not proper solution , please connect with any team member.

Reply:
Yeah , I have tired giving display as block and some margin bottom , everything working fine now.